modules/rubiconBidAdapter.js

```diff
@@ -1107,7 +1107,7 @@ export function classifiedAsVideo(bidRequest) {
   // based on whether or not there is a video object defined in the params
   // Given this legacy implementation, other code depends on params.video being defined
 
-  // if it's bidonmultiformat, we don't care of the video object
+  // if it's bidOnMultiFormat, we don't care of the video object
```

Comment:
Interesting the param below this line was already camel case, possible bug @robertrmartinez ?

Reply:
nah its a locally declared variable so its fine

```js
let isBidOnMultiformat = typeof deepAccess(bidRequest, `params.bidonmultiformat`) !== 'undefined';
```